### ISSUE TITLE
Normal theme to light, dark theme too dark(and too blue). Both bad for eyes

### ISSUE THREAD

**Issue by mzso** [sindresorhus/caprine#392]
Hi!

It'd be nice to have something in between that's pleasant for the eyes. The normal theme with the white background is eye-scroching as always, the dark theme is difficult to read. (Also the blue hue is not good for melatonin levels in the evening)

Even better would be if we could just tune the brightness hue with a slider without a need for themes.

**Comment by dusansimic** [comment]
I agree, the dark theme is not really "dark" as you'd expect. I would rather recommend to add a functionality  for custom themes (adding a custom css file). This would make it more customizable.

**Comment by skymakai** [comment]
I like how dark the 'dark' theme is with 'vibrancy' turned off.  However, the text is a gray color, which makes it hard to read on the dark theme.  I'd like the text to be lighter.

**Comment by elvarb** [comment]
The YakYak dark theme is almost perfect, good baseline for a dark theme

https://github.com/yakyak/yakyak

**Comment by kitor** [comment]
I'm missing this (or similar) theme from original m4d - https://userstyles.org/styles/134433/dark-night-facebook-messenger
Indeed ability to just load own css would be great.

**Comment by sindresorhus** [comment]
I come to agree that the current dark mode is too bright and blue too. Happy to accept a pull request that makes it slightly darker and less blue.

**Comment by sindresorhus** [comment]
I'm not going to do anything with the light theme though. It's already enough work to maintain the dark theme.

One way to support custom themes could be to just try reading a certain CSS file in the user's home directory and if it exists, apply it.
